src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/HdcCurfewTimesRepository.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.HdcCurfewTimes
+
+@Repository
+interface HdcCurfewTimesRepository : JpaRepository<HdcCurfewTimes, Long> {
```

Comment:
so might be able to remove this if its not used

Reply:
removed - thank you!